spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -821,6 +821,9 @@ public void testHistoryTable() {
     // rollback the table state to the first snapshot
     table.manageSnapshots().rollbackTo(firstSnapshotId).commit();
     long rollbackTimestamp = Iterables.getLast(table.history()).timestampMillis();
+    assertThat(rollbackTimestamp)
+        .as("Rollback history timestamp should be greater than first snapshot timestamp")
+        .isGreaterThan(firstSnapshotTimestamp);
```

Comment:
should we assert that the `rollbackTimestamp` is equal to the `tableMetadata.lastUpdatedMillis()` for the first commit?